client/src/core/ProgramTypes.res

```diff
@@ -159,6 +160,24 @@ module Sign = {
     | Positive => ""
     | Negative => "-"
     }
+
+  // WHAT I reordered some fns here - I think it'd be nice to consistently have
```

Comment:
I like just after the type as well. We should consistently make either encode or decode first.

Reply:
Unless you have a preference, I'd go with `decode` first (shifts some focus to `backend`)